### PR TITLE
Add nats dashboard

### PR DESCRIPTION
**Pull request by andriisoldatenko**
fix https://github.com/astronomer/issues/issues/1415

**Which issue this PR fixes**:

**Summary of changes**:

**Which images are updated by this PR?**:

- [x] grafana

**Checklist** (required)

Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.

- [x] version.txt is updated in the changed image(s)

If a security scan fails for an unchanged image...

- [x]  a fix has been applied OR...
- [x]  an [issue]() has been created

**Additional notes for your reviewer**:

depend on https://github.com/astronomer/astronomer/pull/754

### CODE REVIEW

File: grafana/include/stan-dash.json [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "",
+  "title": "NATS Streaming Dashboard",
+  "uid": "LbCzljqika",
+  "version": 1
+}
```
Comment (schnie) [inline grafana/include/stan-dash.json:461]:
```suggestion
}

```

File: grafana/include/nats-dash.json [hunk 2]
```diff
@@ -0,0 +1,1033 @@
...
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "browser",
+  "title": "NATS Server Dashboard",
+  "uid": "eDyoRw7Gz",
+  "version": 1
+}
```
Comment (schnie) [inline grafana/include/nats-dash.json:1033]:
```suggestion
}

```

File: elasticsearch/Dockerfile [hunk 3]
```diff
@@ -20,4 +20,4 @@ ARG BUILD_NUMBER=-1
 LABEL io.astronomer.docker=true
 LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 
-RUN yum -y update bind-license
+RUN yum -y update bind-license dbus
```
Comment (schnie) [inline elasticsearch/Dockerfile:23]: Patching a CVE? Should we separate this into a new PR for audit purposes?

Comment (vishwas-astro) [reply elasticsearch/Dockerfile:23]: Yes please. Let's not overload PRs

File: keda/Dockerfile [hunk 4]
```diff
@@ -26,4 +26,6 @@ ENV OPERATOR=/usr/local/bin/keda \
 
 ENTRYPOINT ["/usr/local/bin/entrypoint"]
 
+RUN yum -y update gnutls
```
Comment (vishwas-astro) [inline keda/Dockerfile:29]: @andriisoldatenko This does not look like a NATS change?

File: kibana/Dockerfile [hunk 5]
```diff
@@ -22,5 +22,5 @@ LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 
 # to fix bind-license security issues
 USER root
-RUN yum -y update bind-license
+RUN yum -y update bind-license dbus
```
Comment (vishwas-astro) [inline kibana/Dockerfile:25]: Same as above comment. Not a NATS change.

File: elasticsearch/Dockerfile (hunk 3 above)
Comment (vishwas-astro) [reply elasticsearch/Dockerfile:23]: Please create a separate PR please

Comment (andriisoldatenko) [reply elasticsearch/Dockerfile]: done https://github.com/astronomer/ap-vendor/pull/40

File: keda/Dockerfile (hunk 4 above)
Comment (andriisoldatenko) [reply keda/Dockerfile]: done https://github.com/astronomer/ap-vendor/pull/40

File: kibana/Dockerfile (hunk 5 above)
Comment (andriisoldatenko) [reply kibana/Dockerfile]: done https://github.com/astronomer/ap-vendor/pull/40

**Pull request merged**